docs/Administrator-Guide/arbiter-volumes-and-quorum.md

```diff
@@ -66,21 +66,20 @@ performance.readdir-ahead: on  `
 ```
 
 The arbiter brick will store only the file/directory names  (i.e. the tree structure)
-and extended attributes (metadata) but not any data. i.e. the file size
+and extended attributes (metadata) but not any data, i.e. the file size
 (as shown by `ls -l`) will be zero bytes. It will also store other gluster
-metadata like the .glusterfs folder and its contents.
+metadata like the `.glusterfs` folder and its contents.
 
 _**Note:** Enabling the arbiter feature **automatically** configures_
 _client-quorum to 'auto'. This setting is **not** to be changed._
 
 ## Arbiter brick(s) sizing
 
 Since the arbiter brick does not store file data, its disk usage will be considerably
-less than the other bricks of the replica. The sizing of the brick will depend on
+less than for the other bricks of the replica. The sizing of the brick will depend on
```

Comment:
can ommit 'than' and and 'the' as follows:
"Since the arbiter brick does not store file data, its disk usage will be considerably
less for other bricks of the replica"

Reply:
Thanks for the review.
Your sentence sounds like the other bricks have less disk usage than the arbiter brick. I do not think that is what is meant, or am I misunderstanding? I though the arbiter brick has less disk usage compared to the other bricks which store actual data?

Reply:
My bad.
The right sentence would be:
 "Since the arbiter brick does not store file data, its disk usage will be considerably
lesser than the other bricks of the replica"

Reply:
I rewrote it and used the word `smaller`, as it seem to be the easiest to understand.